src/test/java/com/github/sttk/cliargs/ParseForTest.java

```diff
@@ -0,0 +1,2688 @@
+package com.github.sttk.cliargs;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.fail;
+import org.junit.jupiter.api.Test;
+
+import com.github.sttk.cliargs.annotation.Opt;
+import com.github.sttk.cliargs.CliArgs;
+import com.github.sttk.cliargs.convert.IntConverter.InvalidIntegerFormat;
+import com.github.sttk.cliargs.convert.DoubleConverter.InvalidDoubleFormat;
+import com.github.sttk.exception.ReasonedException;
+import java.math.BigInteger;
+import java.math.BigDecimal;
+import java.util.List;
+
+@SuppressWarnings("missing-explicit-ctor")
+public class ParseForTest {
+
+  @Test
+  void testParseFor_emptyOptionStoreAndNoArgs() {
+    class MyOptions {}
```

Comment:
It is necessary to add reflect configs for this and other `MyOptions` classes to `reflect-config.json`.